packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,89 @@ section: get-started
 import './get-started.css';
 import { Divider, Timestamp } from '@patternfly/react-core';
 
+<Timestamp>October 2024</Timestamp>
+
+## PatternFly 6.0
+The official release of PatternFly 6 is here! It contains all of the previous features from our alpha and beta releases, but also adds everything described in these release highlights.
+
+### PatternFly AI
+
+We're excited to introduce PatternFly AI: our new effort to support and integrate AI into our design system. We've added guidance and resources, contained within a new section of our website:
+
+- [About PatternFly AI](/patternfly-ai/about-patternfly-ai)
+- [AI guidelines](/patternfly-ai/ai-guidelines)
+- [Chatbot](/patternfly-ai/chatbot/about-chatbot)
+- [Conversation design](/patternfly-ai/conversation-design)
+
+As with all things AI currently, this area will continue to grow and mature with time.
+
+### Component groups restructuring
+
+We made many updates to our component groups extension, to improve accuracy, usability, and alignment with PatternFly 6. We've moved its website section to our top-level navigation for better visibility, and also to support necessary sub-navigation. We've conceptually grouped the components into functional categories. Additionally, we renamed some of the components to be more accurate. 
+
+[Check out the updated documentation](/component-groups/about-component-groups), which includes:
+
+| **Category** | **Components** | 
+| --- | --- |
+| Content containers | - Details page <br /> - Multi-content card<br /> - Page header (previously named "content header")<br /> - Service card (new!) |
+| Controls | - Bulk select <br /> - Close button <br /> - Responsive actions (new!) |
+| Error communication | - Error boundary <br /> - Error state <br /> - Missing page (previously named "invalid object") <br /> - Unauthorized access (previously named "not authorized") <br /> - Unavailable content <br /> - Warning modal |
+| Helpers | - Column management modal <br /> - Log snippet <br /> - Shortcut grid |
+| Status and state indicators | - Ansible <br /> - Severity (previously Battery) <br /> - Skeleton table <br /> - Status (new!) <br /> - Tag count |
+
+### React component updates 
+
+Here are the most significant updates we made to our React components: 
+
+- Data list
+    - Refactored `<DataListCheck>` to use `<Checkbox>` internally. As a result, `id` is now a required prop for ` <DataListCheck>`. This also fixes broken checkboxes in the examples. 
+- Dropdown
```

Comment:
So we kind of reverted this.  We have a prop to set the time out but it is `0` by default.  We added some other logic internally to stop the scroll from jumping when focusing the first item in the menu.

Reply:
ahh okay, is that `focusTimeoutDelay`? How's this

> - Dropdown
>     - Added `focusTimeoutDelay`, which specifies the time to wait before setting focus on the first dropdown item when `shouldFocusFirstItemOnOpen` is set. 
>     - Fixed issues with invalid and jumpy scrolling when focusing on the first menu item.

Reply:
Sounds good @edonehoo